schemas/workflowexecution.schema.tpl.json

```diff
@@ -0,0 +1,22 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/WorkflowExecution",
```

Comment:
note: we use also lowerCamelCase for the file names.

Reply:
?